client/src/components/widgets/VTKPlot/template.html

```diff
@@ -1,4 +1,33 @@
 <div ref="vtk" class="plot">
+  <v-menu
+    v-if="enableRangeTooltip"
+    open-on-hover
+    offset-y
+    left
+    top
+    :close-on-content-click="false"
+    min-width="fit-content"
+    height="fit-content"
+  >
+    <template v-slot:activator="{ on }">
+      <v-icon v-on="on" :style="{position: 'absolute', right: '0', bottom: '0'}"
+        >mdi-information-variant</v-icon
+      >
+    </template>
+    <v-list>
+      <v-list-item dense v-if="!xaxisVisible">
+        <v-list-item-title>X Range: {{tooltipText().x}}</v-list-item-title>
+      </v-list-item>
+      <v-list-item dense v-if="!yaxisVisible">
+        <v-list-item-title>Y Range: {{tooltipText().y}}</v-list-item-title>
+      </v-list-item>
+      <v-list-item dense v-if="!showScalarBar">
+        <v-list-item-title
+          >Scalar Range: {{tooltipText().scalar}}</v-list-item-title
```

Comment:
weird indentation of `>`

Reply:
It seems that autoflake makes some interesting choices if there is not a space after/before the tag... Found a few more instances of this as well. Cleaned up now.